geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/membership/gms/messenger/JGroupsMessengerJUnitTest.java

```diff
@@ -484,6 +498,38 @@ public void testBigMessageIsFragmentedWhenBroadcast() throws Exception {
     doTestBigMessageIsFragmented(false, true);
   }
 
+  public static class ByteHolder implements DataSerializableFixedID {
+    static final int DSFID = 123456;
+    private byte[] payload;
+
+    @Override
+    public int getDSFID() {
+      return DSFID;
+    }
+
+    public ByteHolder(byte[] payload) {
+      this.payload = payload;
+    }
+
+    public ByteHolder() {}
```

Comment:
Default constructor isn't ever used (IntelliJ warning).

Reply:
I'll address the warning - the default constructor is necessary for deserialization.  The constructor is registered with the serializer.
The test will fail if we remove that constructor.